content/2024/06/maxmind-appoints-seasoned-data-science-leader-rupert-young-as-chief-product-officer.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "MaxMind appoints seasoned data science leader Rupert Young as Chief Product Officer"
+date: "2024-06-11"
+images:
+  - /images/2024/06/rupert_blog.png
```

Comment:
This file itself is pretty large (665KB), is there a way to compress it our convert the file type?

Reply:
I'll check with the team!

Reply:
We'll compress and update shortly, much appreciated!

Reply:
Unrelated, but it may also be worth compressing `opmc-maxmind-ecommerce-plugin-partnership.png`, which is 6.23MB.

Reply:
this blog post image has been resized, and I've passed along the other image resizing request. Thanks @dhogan8

Reply:
OMPC image has been compressed, I'll create a separate PR for it sometime today and will swap out the image.